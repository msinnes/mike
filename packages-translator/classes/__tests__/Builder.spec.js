const Builder = require('../Builder');

describe('Builder', () => {
  it('should be a static class', () => {
    expect(() => {
      Builder('string');
    }).toThrowErrorMatchingSnapshot();

    expect(() => {
      new Builder('string');
    }).toThrowErrorMatchingSnapshot();
  });
});