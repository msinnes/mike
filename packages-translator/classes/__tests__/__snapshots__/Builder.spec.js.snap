// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Builder should be a static class 1`] = `"ClassError: Cannot call a class as a function"`;

exports[`Builder should be a static class 2`] = `"ClassError: Static classes cannot be instantiated"`;
